test/prototype_transforms_dispatcher_infos.py

```diff
@@ -63,17 +63,40 @@ def sample_inputs(self, *feature_types, filter_metadata=True):
                     yield args_kwargs
 
 
-def xfail_python_scalar_arg_jit(name, *, reason=None):
+def xfail_jit_python_scalar_arg(name, *, reason=None):
```

Comment:
Just a rename that moves the `jit` term to the front of the name to make it clear this is only a JIT issue.